client/my-sites/stats/stats-list/stats-list-card.jsx

```diff
@@ -82,18 +83,43 @@ const StatsListCard = ( {
 		);
 	};
 
+	const getLabelIcon = ( labelIconType ) => {
+		if ( labelIconType === 'folder' ) {
+			return <Icon className="stats-icon" icon={ file } size={ 22 } />;
+		} else if ( labelIconType === 'tag' ) {
+			return <Icon className="stats-icon" icon={ tag } size={ 22 } />;
+		}
+	};
+
+	const outputLeftItem = ( item ) => {
```

Comment:
Wondering if we could use a better name for the function, because it doesn't output anything IMO.

Reply:
It generates JSX for the left side of the label. I named it this way to match the right side. Do you think that `generateLeftItem` and `generateRightItem` would be better?

Reply:
`get` would do too. Either works for me.